<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admf</class>
 <widget class="QWidget" name="admf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddUser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="ulbl">
           <property name="text">
            <string>&amp;Login</string>
           </property>
           <property name="buddy">
            <cstring>llnEd</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="llnEd"/>
         </item>
         <item>
          <widget class="QLabel" name="pslbl">
           <property name="text">
            <string>&amp;Pass</string>
           </property>
           <property name="buddy">
            <cstring>passlne</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passlne"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="lv">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addUbt">
         <property name="text">
          <string>Add User</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remUsBt">
         <property name="text">
          <string>Remove User</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
